2019-02-28 17:38:49,463 - public_methods.py[line:66] - INFO: 获取ID时报错，错误为:<urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>,url为:https://10.82.2.29:7144/api/v1.0/one/dsl/query?dsl={"Salesman":{"fields":["id","PersonnelNumber"],"cond":{"in":{"PersonnelNumber":["0041","0118","02824","05160","06040","123456","123457","22370"]}},"limit":500}}
2019-02-28 17:38:49,463 - public_methods.py[line:66] - INFO: 获取ID时报错，错误为:<urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>,url为:https://10.82.2.29:7144/api/v1.0/one/dsl/query?dsl={"Salesman":{"fields":["id","PersonnelNumber"],"cond":{"in":{"PersonnelNumber":["0041","0118","02824","05160","06040","123456","123457","22370"]}},"limit":500}}
2019-02-28 17:38:49,463 - public_methods.py[line:66] - INFO: 获取ID时报错，错误为:<urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>,url为:https://10.82.2.29:7144/api/v1.0/one/dsl/query?dsl={"Salesman":{"fields":["id","PersonnelNumber"],"cond":{"in":{"PersonnelNumber":["0041","0118","02824","05160","06040","123456","123457","22370"]}},"limit":500}}
2019-02-28 17:38:49,463 - public_methods.py[line:66] - INFO: 获取ID时报错，错误为:<urlopen error [WinError 10061] 由于目标计算机积极拒绝，无法连接。>,url为:https://10.82.2.29:7144/api/v1.0/one/dsl/query?dsl={"Salesman":{"fields":["id","PersonnelNumber"],"cond":{"in":{"PersonnelNumber":["0041","0118","02824","05160","06040","123456","123457","22370"]}},"limit":500}}
